src/bot/handlers/tasks.py

```diff
@@ -72,9 +72,8 @@ async def back_subcategory_callback(update: Update, context: ContextTypes.DEFAUL
 @logger_decor
 async def view_task_callback(update: Update, context: CallbackContext, limit: int = 3):
     task_service = TaskService()
-    page_number = context.user_data.get("page_number", 1)
-    offset = (page_number - 1) * limit
-    tasks_to_show = await task_service.get_user_tasks_by_page(limit, offset)
+    user_data = context.user_data
+    tasks_to_show, offset, page_number = await task_service.get_user_tasks_by_page(user_data, limit)
```

Comment:
Зачем сюда user_data передавать?
```suggestion
    tasks_to_show, offset, page_number = await task_service.get_user_tasks_by_page(user_data.get("page_number", 1), limit)
```